ts/main.ts

```diff
@@ -0,0 +1,117 @@
+/* global data */
+const $row = document.querySelector('.movies-row') as HTMLElement;
+
+if (!$row) throw new Error('$row not found.');
+
+async function getMovies(): Promise<void> {
+  try {
+    const options = {
+      method: 'GET',
+      headers: {
+        accept: 'application/json',
+        Authorization:
+          'Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiI1ZDZjMTEyZWEwOTg2N2Q4MmJjMzNmMTc0YzZjNjkyMSIsInN1YiI6IjY1YTAzN2I4NzI2ZmIxMDEyYmY4YWY5ZCIsInNjb3BlcyI6WyJhcGlfcmVhZCJdLCJ2ZXJzaW9uIjoxfQ.CU2LQtrnjr4YUnU4bl7n9bFGYwY9XJOsSiyISbpsDcs',
+      },
+    };
+
+    const response = await fetch(
+      'https://api.themoviedb.org/3/movie/top_rated?language=en-US&page=1',
+      options,
+    );
+
+    if (!response.ok) throw new Error('Failed to get a response.');
+
+    const responseData = await response.json();
+    const resultsArr = responseData.results as Movie[];
+
+    // Render each movie to DOM
+    for (let i = 0; i < resultsArr.length; i++) {
+      resultsArr[i].entryId = data.nextEntryId;
+      const movieElement = renderCard(resultsArr[i]);
+      $row.appendChild(movieElement);
+      data.nextEntryId++;
+    }
+  } catch (error) {
+    console.log(`Error: ${error}`);
+  }
+}
+
+function renderCard(data: Movie): HTMLElement {
+  const $outerColumn = document.createElement('div');
+  $outerColumn.setAttribute('class', 'column-fourth');
+  $outerColumn.setAttribute('data-entry-id', data.entryId.toString());
+
+  const $cardDivElement = document.createElement('div');
+  $cardDivElement.setAttribute('class', 'card');
+
+  $outerColumn.appendChild($cardDivElement);
+
+  const $cardImage = document.createElement('img');
+  $cardImage.setAttribute('class', 'card-img');
+  $cardImage.setAttribute(
+    'src',
+    'http://image.tmdb.org/t/p/w500/' + data.poster_path,
+  );
+  $cardImage.setAttribute('alt', data.title);
+
```

Comment:
I'm glad you thought to add the `alt` attribute here! You should make it a little more specific though like 'data.title' + 'Movie Poster' so that it ends up something like 'The Dark Knight Movie Poster' and is more descriptive